spec.md

```diff
@@ -517,6 +641,47 @@ To delete a blob, perform a `DELETE` request to a path in the following format:
 Upon success, the registry MUST respond with code `202 Accepted`.
 If the blob is not found, a `404 Not Found` code MUST be returned.
 
+### Backwards Compatibility
+
+Client implementations MUST support registries that implement partial or older versions of the OCI Distribution Spec.
+This section describes client fallback procedures that MUST be implemented when a new/optional API is not available from a Registry.
```

Comment:
right, so, if there is a backward-compat condition, then is it _required_ that this feature bumps the version to 1.1?